game/main.css

```diff
@@ -6,60 +6,74 @@ body {
     border-collapse: collapse;
     background: url(game_icons/board-game-bottom.png) no-repeat rgb(181, 228, 255);
     background-position: bottom;
-    background-position-y: 115%;
-    width: 820px;
-    height: 758px;
+    background-size: 100%;
+    width: 602px;
+    height: 540px;
 }
 
 table tr td {
-    width: 80px;
-    height: 80px;  
+    width: 60px;
+    height: 60px;  
 }
 
 .active {
```

Comment:
Kotek fruwa jakieś 10px nad ziemią. Wynika to z faktu, że w miejscu w którym znajdują się jego nogi kończy się tabela. Masz do wyboru:
1. zmniejszyć wysokość całego pola gry i podnieść "ziemię w tle do góry"
2. wysunąć kotka z tła w dół poza komórkę tabeli.
Pierwsze będzie w moim odczuciu łatwiejsze.

Reply:
Dotyczy rozdzielczości 1920x1080px(full hd). Dla rozdzielczości 2048x1152 jest ok.

Reply:
![cotek2](https://user-images.githubusercontent.com/37185264/54359461-ff91df00-4662-11e9-8389-542ae10683a4.png)

Reply:
u mnie jest ok:
![image](https://user-images.githubusercontent.com/29073258/54397648-7658c780-46b7-11e9-959e-077d437add07.png)